pages/linux/rdesktop.md

```diff
@@ -0,0 +1,28 @@
+# RDESKTOP
```

Comment:
please use lowercase for the header -- in most cases this corresponds to the command itself.